src/TLuaInterpreter.cpp

```diff
@@ -3129,9 +3043,8 @@ int TLuaInterpreter::loadMap(lua_State* L)
                             luaL_typename(L, 1));
             lua_error(L);
             return 1;
-        } else {
-            location = QString::fromUtf8(lua_tostring(L, 1));
         }
+        location = QString::fromUtf8(lua_tostring(L, 1));
```

Comment:
```suggestion
        QString location = QString::fromUtf8(lua_tostring(L, 1));
```
Then you can again delete the `QString location;` line above as well.

Reply:
Nevermind, things aren't that easy here.